engine/events/halloffame.asm

```diff
@@ -15,7 +15,9 @@ HallOfFame::
 	set STATUSFLAGS_HALL_OF_FAME_F, [hl]
 
 ; Bug: Gold/Silver fail to (conditionally) erase the previous save and
-; initialize the current save, if the player did not save on this playthrough!
+; initialize the current save, if the player did not save on this playthrough.
+; pokecrystal does the following here (routine doesn't exist in Gold/Silver):
```

Comment:
Remove lines 19 and 20; they're suitable for a bugs doc, along with the `HallOfFame_InitSaveIfNeeded` code.

Reply:
I assume it makes sense to move the comment to the top of the routine then as well? (current placement was because that's where the missing needed line is in pokecrystal)

Reply:
It's fine where it is.